presto/README.md

```diff
@@ -0,0 +1,28 @@
+# Presto
+
+Presto is a distributed SQL query engine for big data.
+- [Github](https://github.com/prestodb/presto)
+- [Homepage](https://prestodb.io)
+
+The benchmarks are based on Presto version `0.287`.
+
+We assume that a Presto cluster is already running. For more information, visit [Getting Started](https://prestodb.io/getting-started/).
+
+----------
+## Steps
+
+1. Download the parquet file and upload it to an S3 Bucket ex. s3://your-bucket/clickbench-parquet/hits/hits.parquet.
```

Comment:
We already have it,

`https://clickhouse-public-datasets.s3.eu-central-1.amazonaws.com/hits_compatible/athena_partitioned/hits_{0..99}.parquet')`

Reply:
Okay, made a note of this in the README.